Procfile

```diff
@@ -1 +1,2 @@
-web: babel-node bin/server.js
+web:            babel-node bin/server.js server
+expireSentry:   babel-node bin/server.js expire-sentry
```

Comment:
I hope so... Because it's what I do in the queue...

I haven't really looked at the logs so I'm not 100% sure it works :)
well, it been a while since I've seen the logs...they are hard to find because they are prefixed heroku scheduler something...